welcome/learn_more_about_openshift.adoc

```diff
@@ -30,44 +31,184 @@ Use the following sections to find content to help you learn about and use {prod
 |
 | xref:../backup_and_restore/index.adoc#backup-restore-overview[Backup and restore]
 |
+
+| link:https://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[OpenShift Interactive Learning Portal]
+|
+| xref:../support/getting-support.adoc#getting-support[Getting Support] and | link:https://access.redhat.com/articles/4217411[OpenShift Knowledgebase articles]
```

Comment:
The link to kbase articles isn't showing up in the preview build so it just reads "Getting Support and"